### PR TITLE
Contracts v2

### PR DESCRIPTION
**Pull request by raymogg**
# Motivation
With the updates contracts, the executioner needed some upgrading

# Changes
- includes changes for removing serialisation and using tracer-utils
- updates trader contract abi
- updates types and function calls for contracts-v2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 let express = require("express")
 let Web3 = require('web3');
-let trader = require('./contracts/Trader.json')
+let traderABI = require('./contracts/Trader.json')
```
Comment (dospore) [inline index.js:3]: if you want you can install @tracer-protocol/contracts to avoid this in the future

File: test/test.js [hunk 2]
```diff
@@ -10,46 +10,39 @@ require("../index")
...
-    "market": "0x529da3408a37a91c8154c64f3628db4eaa7b8da2",
+    "user": "0x529da3408a37a91c8154c64f3628db4eaa7b8da2",
+    "target_tracer": "0x529da3408a37a91c8154c64f3628db4eaa7b8da2",
     "side": "Bid",
     "price": 12,
     "amount": 5,
     "expiration": 1596600983,
-    "flags": { "bits": 1 },
     "signed_data": web3.utils.hexToBytes(exampleSignatureRaw),
     "nonce": "0x44"
```
Comment (dospore) [inline test/test.js:20]: did we remove nonce??

**Review by dospore — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (raymogg) [reply index.js:3]: I'll set that up on a new branch ty @dospore.

File: test/test.js (hunk 2 above)
Comment (raymogg) [reply test/test.js:20]: Ah yeah good point I'll remove all references to nonce

**Pull request merged**